reconcile/utils/jjb_client.py

```
@@ -275,7 +275,9 @@ def update(self) -> None:
 
     @staticmethod
     def get_jjb(args: Iterable[str]) -> Any:
-        from jenkins_jobs.cli.entry import JenkinsJobs  # noqa: PLC0415
+        from jenkins_jobs.cli.entry import (  # ruff: ignore[import-outside-top-level]
```

Ruff (0.16.0)

[warning] Invalid rule code in suppression: import-outside-top-level

Enable `lint.preview` to use rule names

(RUF102)